cscope.py

```diff
@@ -170,10 +187,32 @@ class CscopeVisiter(sublime_plugin.TextCommand):
     def __init__(self, view):
         self.view = view
 
-    def run(self, edit):
+    def run(self, edit, **args):
         if self.view.settings().get('syntax') == CSCOPE_SYNTAX_FILE:
+            if args.get('direction'):
+                d = args.get('direction')
+                r = self.view.get_regions('results')
+                if r == []:
+                    return
+                view_height = len(r) - 1
+                s = self.view.settings().get('cscope_results_sel')
+                what = {
+                    'up': -1,
+                    'down': 1,
+                    'up2': -20,
+                    'down2': 20
+                }
+                if d not in what:
+                    return
+                s = s + what[d]
+                if s < 0: s = 0
+                if s > view_height: s = view_height
+
+                update_result_selection(self.view, s, r[s])
+                return
+
             root_re = re.compile(r'In folder (.+)')
-            filepath_re = re.compile(r'^(.+):$')
+            filepath_re = re.compile(r'^(((?!scope: ).)+):$')
```

Comment:
Can you explain this change please? Why the change in regexp?

Reply:
If line with symbol ends with ':' it is catched by this regex (seen below), which leads to such error when trying to open results lower than that.
```
Unable to open file: E:\Dev\C\sandbox\cscope_tmp\    17 [scope: main] case ENUM_TESTCASE_1
```